<launch>

    <!-- Robot namespace -->
    <arg name="robot_name" default="my_gen3"/>
    <arg name="start_delay_seconds" default="0"/>

    <!-- Launch the example in that namespace -->
    <node name="full_arm_movement_cpp" pkg="kortex_examples" type="example_full_arm_movement_cpp" output="screen" launch-prefix="bash -c 'sleep $(arg start_delay_seconds); $0 $@' ">
        <param name="robot_name" value="$(arg robot_name)"/>
    </node>

</launch>